src/ray/raylet/reconstruction_policy_test.cc

```diff
@@ -66,6 +66,10 @@ class MockObjectDirectory : public ObjectDirectoryInterface {
   std::string DebugString() const override { return ""; }
 
   MOCK_METHOD0(GetLocalClientID, ray::ClientID());
+#ifdef __clang__
```

Comment:
These should be put into compiler flags in bazel, not into the source files. It would be even better if the warning is fixed.

Reply:
I didn't put them into `.bazelrc` because normally we can and do fix this error. It's just that in this particular instance it's impractical to, since `MOCK_CONST_METHOD0` etc. are defined in headers we don't control.